.github/ACKNOWLEDGMENTS.md

```diff
@@ -45,3 +45,5 @@
 * [Timjan Kalajdzievski](https://github.com/timjank) (Xanadu) - :beverage_box: Beard Czar
 
 * [Yuan Yao](https://github.com/sylviemonet) (Télécom Paris) - :dog: Schrödinger's Dog's owner
+
+* [Ali Asadi](https://github.com/maliasadi) (Western University)
```

Comment:
It would be nice if you also picked a "title" and emoji for yourself. If you prefer not, it is is course OK.

Reply:
I wasn't sure to add mine there. Done! :smile:

Reply:
Are you sure you added it?